app/src/main/java/com/locoquest/app/MainActivity.kt

```diff
@@ -81,12 +81,12 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
      * @param permissions An array of permission strings.
      * @param grantResults An array of grant results for the corresponding permissions.
      */
-    override fun onRequestPermissionsResult(
+    override fun onPermissionRequestCompletion(
```

Comment:
I edited the name of this method to better reflect what it is doing. The method is specifically called when a permission request is completed, not when it yields a result. It's a bit nit picky but I just wanted the method to be clear! 

https://kotlinlang.org/docs/coding-conventions.html#choose-good-names

Reply:
onRequestPermissionsResult is the function name commonly used in this case.